config/rubocop-2.5.yaml

```diff
@@ -0,0 +1,1366 @@
+require:
+  - rubocop-rails
+  - rubocop-performance
+
+AllCops:
+  DisabledByDefault: false
+  Exclude:
+    - 'vendor/**/*'
+    - 'tmp/**/*'
+  TargetRubyVersion: 2.5
```

Comment:
Pretty much the only change + https://github.com/bespokepost/code-climate-config/pull/7/commits/63bda3bc8598f52d3d55c1a5558be3fc8107ac02